files/en-us/tools/tools_toolbox/index.html

```diff
@@ -82,7 +82,7 @@ <h3 id="Toolbox_controls">Toolbox controls</h3>
  <li>close the window</li>
  <li>toggle the window between attached to the bottom of the browser window, and attached to the side of the browser window</li>
  <li>toggle the window between standalone and attached to the browser window</li>
- <li>access <a href="#settings">developer tool settings</a></li>
+ <li>access <a href="/en-US/docs/Tools/Settings">developer tool settings</a></li>
 </ul>
 
 <h2 id="Settings_2">Settings</h2>
```

Comment:
```suggestion
<h2 id="settings">Settings</h2>
```
Since we're here anyway